src/components/Items/ProfileContact.js

```diff
@@ -3,7 +3,7 @@ import Mi from 'react-native-vector-icons/MaterialIcons';
 import PropTypes from 'prop-types';
 import {Image, Pressable, StyleSheet, Text, View} from 'react-native';
 
-import MemoIc_Checklist from '../../assets/icons/Ic_Checklist';
+import IconChecklist from '../../assets/icons/Ic_Checklist';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use the renamed `IconChecklist` component instead of `MemoIc_Checklist`. This change is fine as long as the exported component from `../../assets/icons/Ic_Checklist` is indeed renamed to `IconChecklist`.
```diff
- import MemoIc_Checklist from '../../assets/icons/Ic_Checklist';
+ import IconChecklist from '../../assets/icons/Ic_Checklist';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->